en/ros2/index.md

```diff
@@ -1,38 +1,43 @@
 # ROS 2
 
-[ROS 2](https://docs.ros.org/en/humble/#) is the newest version of [ROS](http://www.ros.org/) (Robot Operating System), a general purpose robotics library that can be used with the PX4 Autopilot to create powerful drone applications.
-It captures most of the learnings and features of [ROS 1](../ros/ros1.md), improving a number of flaws of the earlier version.
+[ROS 2](https://docs.ros.org/en/humble/#) is a powerful general purpose robotics library that can be used with the PX4 Autopilot to create powerful drone applications.
 
 :::warning Tip
 The PX4 development team highly recommend that you use/migrate to this version of ROS!
+
+This is the newest version of [ROS](http://www.ros.org/) (Robot Operating System).
+It significantly improves on ROS "1", and in particular allows a much deeper and lower-latency integration with PX4.
 :::
 
+ROS benefits from an active ecosystem of developers solving common robotics problems, and access to other software libraries written for Linux.
+It can be used, for example, for [computer vision](../computer_vision/index.md) solutions.
+
+ROS 2 enables a very deep integration with PX4, to the extent that you can create flight modes in the ROS2 that are indistinguisable from internal PX4 modes, and directly read from and write to internal uORB topics at high rate.
+It is recommended (in particular) for control and communication from a companion computer where low latency is important, when leveraging existing libraries from Linux, or when writing new high level flight modes.
+
 Communication between ROS 2 and PX4 uses middleware that implements the [XRCE-DDS protocol](../middleware/uxrce_dds.md).
 This middleware exposes PX4 [uORB messages](../msg_docs/index.md) as ROS 2 messages and types, effectively allowing direct access to PX4 from ROS 2 workflows and nodes.
 The middleware uses uORB message definitions to generate code to serialise and deserialise the messages heading in and out of PX4.
 These same message definitions are used in ROS 2 applications to allow the messages to be interpreted.
 
-To use the [ROS 2](../ros2/user_guide.md) over XRCE-DDS effectively, you must (at time of writing) have a reasonable understanding of the PX4 internal architecture and conventions, which differ from those used by ROS.
-In the near term future we plan to provide ROS 2 APIs to abstract PX4 conventions, along with examples demonstrating their use.
-
-The main topics in this section are:
-- [ROS 2 User Guide](../ros2/user_guide.md): A PX4-centric overview of ROS 2, covering installation, setup, and how to build ROS 2 applications that communicate with PX4.
-- [ROS 2 Offboard Control Example](../ros2/offboard_control.md)
-
 ::: info
-ROS 2 is officially supported only on Linux platforms.
-Ubuntu 20.04 LTS is the official supported distribution.
```

Comment:
FYI decided not to mention this here. That way we can maintain support info that we want in the guide.